src/System.CommandLine/Argument.cs

```diff
@@ -119,6 +135,9 @@ public List<ISuggestionSource> Suggestions
             }
         }
 
+        /// <summary>
+        /// Gets or sets the type of the argument.
```

Comment:
Maybe something like `Gets or sets the <see cref="System.Type" /> that the argument will be converted to.`?

Reply:
I like it - and I just learned something new.